dataservice/api/biospecimen_diagnosis/resources.py

```diff
@@ -0,0 +1,164 @@
+from flask import abort, request
+from marshmallow import ValidationError
+from webargs.flaskparser import use_args
+
+from dataservice.extensions import db
+from dataservice.api.common.pagination import paginated, Pagination
+from dataservice.api.biospecimen_diagnosis.models import (
+    BiospecimenDiagnosis
+)
+from dataservice.api.biospecimen_diagnosis.schemas import (
+    BiospecimenDiagnosisSchema
+)
+from dataservice.api.common.views import CRUDView
+from dataservice.api.common.schemas import filter_schema_factory
+
+
+class BiospecimenDiagnosisListAPI(CRUDView):
```

Comment:
We really need to move away from having the user manage these associations via individual requests.
Would it be too much to try that here?